src/Unleash/IUnleash.cs

```diff
@@ -68,21 +61,6 @@ public interface IUnleash : IDisposable
         /// <returns>A weighted variant or the supplied default value if feature is not available</returns>
         Variant GetVariant(string toggleName, UnleashContext context, Variant defaultValue);
 
-        /// <summary>
-        /// Get available feature variants.
-        /// </summary>
-        /// <param name="toggleName">The name of the toggle.</param>
-        /// <returns>A list of available variants.</returns>
-        IEnumerable<VariantDefinition> GetVariants(string toggleName);
-
-        /// <summary>
-        /// Get available feature variants.
-        /// </summary>
-        /// <param name="toggleName">The name of the toggle.</param>
-        /// /// <param name="context">The Unleash context to evaluate the toggle state against.</param>
-        /// <returns>A list of available variants.</returns>
-        IEnumerable<VariantDefinition> GetVariants(string toggleName, UnleashContext context);
```

Comment:
These aren't in the documentation either, but also warrants a major release

Reply:
Yep, same as https://github.com/Unleash/unleash-client-dotnet/pull/224/files#r1661225276